governance.md

```diff
@@ -0,0 +1,256 @@
+
+* evangelising about the project (e.g. a link on a website and word-of-mouth awareness raising)
+* informing developers of strengths and weaknesses from a new user perspective
+* providing moral support (a ‘thank you’ goes a long way)
+* providing financial support (the software is open source, but its developers need to eat)
+
+Users who continue to engage with the project and its community will often become more and more
+involved. Such users may find themselves becoming contributors, as described in the next section. 
+
+### Contributors
+
+Contributors are community members who contribute in concrete ways to the project. Anyone can become
+a contributor, and contributions can take many forms, as detailed in a separate document. 
+There is no expectation of commitment to the project, no specific skill requirements and no selection process.
+
+In addition to their actions as users, contributors may also find themselves doing one or more of the following:
+
+*  supporting new users (existing users are often the best people to support new users)
+*  reporting bugs
+*  identifying requirements
+*  providing graphics and web design
+*  programming
+*  assisting with project infrastructure
+*  writing documentation
+*  fixing bugs
+*  adding features
+
+Contributors engage with the project through the issue tracker and mailing list, or by writing or
+editing documentation. They submit changes to the project itself via patches, which will be
+considered for inclusion in the project by existing committers (see next section). The developer
+mailing list is the most appropriate place to ask for help when making that first contribution.
+
+As contributors gain experience and familiarity with the project, their profile within, and
+commitment to, the community will increase. At some stage, they may find themselves being nominated
+for committership. 
+
+
+### Commiters
+
+Committers are community members who have shown that they are committed to the continued development
+of the project through ongoing engagement with the community. 
+Committership allows contributors to more easily carry on with their project related activities by
+giving them direct access to the project’s resources. 
+That is, they can make changes directly to project outputs, without having to submit changes via patches.
+
+This does not mean that a committer is free to do what they want. In fact, committers have no more
+authority over the project than contributors.  While committership indicates a valued member of the
+community who has demonstrated a healthy respect for the project’s aims and objectives, their work
+continues to be reviewed by the community before acceptance in an official release. The key
+difference between a committer and a contributor is when this approval is sought from the community.
+A committer seeks approval after the contribution is made, rather than before.
```

Comment:
Hmm. I think a committer's "default mode" should still be to seek approval before making contributions. 

IMHO, at least the workflow should be the same: a committer should still open PRs and be open to feedback. The difference being that you can overrule and still merge even if there are concerns from the community.

Reply:
I was thinking about this as well, agree with you. It should always be reviewed by another person (4-eyes-principle)